test_alice_locks_tokens & test_velock_whitelisted are failing

### How to reproduce

They just showed up in CI. [Here's the log](https://github.com/oceanprotocol/df-py/actions/runs/4955589891/jobs/8865119917).

I did _not_ have these failures yesterday. (In fact I'd got test_alice_locks_tokens passing in #569.) Maybe these are sporadic? Maybe it's barge-related?

### Traceback: test_alice_locks_tokens

```text
...
___________________________ test_alice_locks_tokens ____________________________

    @enforce_types
    def test_alice_locks_tokens():
        """Lock tokens then check balance."""
        veOCEAN.checkpoint({"from": alice})
        OCEAN.approve(veOCEAN.address, TA, {"from": alice})
    
        t0 = chain.time()
        t1 = t0 // WEEK * WEEK + WEEK
        t2 = t1 + YEAR
        chain.sleep(t1 - t0)
        chain.mine()
    
        assert OCEAN.balanceOf(alice) != 0
    
        veOCEAN.create_lock(TA, t2, {"from": alice})
    
        assert OCEAN.balanceOf(alice) == 0
    
        epoch = veOCEAN.user_point_epoch(alice)
        assert epoch != 0
    
        assert veOCEAN.get_last_user_slope(alice) != 0
>       aliceVotingPower = (veOCEAN.balanceOf(alice, chain.time())) / toBase18(1.0)
E       brownie.exceptions.VirtualMachineError: revert

util/test/veOcean/test_lock.py:45: VirtualMachineError
---------------------------- Captured stdout setup -----------------------------
mnemonic: 'usual body render message material high country group seed eight insect that'
mnemonic: 'weird slot good beauty develop strike member benefit taxi escape fly skate'
```

### Traceback: test_velock_whitelisted

```text
...
___________________________ test_velock_whitelisted ____________________________

    @enforce_types
    def test_velock_whitelisted():
        """Test that a whitelisted contract can create a lock."""
    
        # Assert that the contract is whitelisted
        smartWalletChecker.setAllowedContract(veLocker, True, {"from": deployer})
        assert smartWalletChecker.check(veLocker) == True
    
        t0 = chain.time()
        t1 = t0 // WEEK * WEEK + WEEK
        t2 = t1 + WEEK
        chain.sleep(t1 - t0)
    
        assert OCEAN.balanceOf(veLocker) != 0
        veLocker.create_lock(TA, t2, {"from": deployer})
        assert OCEAN.balanceOf(veLocker) == 0
    
        epoch = veOCEAN.user_point_epoch(veLocker)
        assert epoch != 0
    
        assert veOCEAN.get_last_user_slope(veLocker) != 0
>       veLockerVotingPower = (veOCEAN.balanceOf(veLocker, chain.time())) / toBase18(1.0)
E       brownie.exceptions.VirtualMachineError: revert

util/test/veOcean/test_whitelist_sc.py:52: VirtualMachineError
```

--- a/util/test/veOcean/test_lock.py
+++ b/util/test/veOcean/test_lock.py
@@ -22,7 +22,6 @@ TA = to_wei(10.0)
 
 
 @enforce_types
-@pytest.mark.skip(reason="unskip once #575 fixed")
 def test_alice_locks_tokens():
     """Lock tokens then check balance."""
     veOCEAN.checkpoint({"from": alice})

--- a/util/test/veOcean/test_whitelist_sc.py
+++ b/util/test/veOcean/test_whitelist_sc.py
@@ -3,7 +3,6 @@ from enforce_typing import enforce_types
 import pytest
 from pytest import approx
 
-
 from util import networkutil, oceanutil
 from util.constants import BROWNIE_PROJECT as B
 from util.base18 import to_wei

